packages/api/src/errors.js

```diff
@@ -58,6 +58,15 @@ export class TokenNotFoundError extends HTTPError {
 }
 TokenNotFoundError.CODE = 'ERROR_TOKEN_NOT_FOUND'
 
+export class TokenBlockedError extends HTTPError {
+  constructor (msg = 'API token is blocked') {
```

Comment:
@dchoi27 do we want to add something like `...please contact support@web3.storage` to the blocked token error message in case we ever block folks who are accidentally causing us problems rather deliberately ?

Reply:
probably.  we're going to add that verbiage in the UI soon so may as well return it in the API response.  @e-schneid can you add that?

Reply:
updated